docs/src/developer/reference/config-options.md

```diff
@@ -764,4 +764,35 @@ Example:
 multiIngress: 
    red.example.com: https://accounts.red.example.com/conversation-join/
    green.example.com: https://accounts.green.example.net/conversation-join/
-```
\ No newline at end of file
+```
+
+### Webapp
+
+In order to completely hide the root domain to the webapp, some environment variable need to be set. They will allow the webapp hostname to be used to generate the API endpoint, team settings links, account page links and CSP headers. 
```

Comment:
IMHO this one is still relevant: *hostname* can mean very different things in the context of Wire (hostname of the pod, the node, the `Host` header ... ;) ).

Reply:
oh yes, it is relevant; for sure. I has been auto "resolved" because the file was edited, but I was planning on changing that (just working on some other topics right now). 
This is in my todo :)